datadog_checks_dev/datadog_checks/dev/tooling/codeowners.py

```diff
@@ -0,0 +1,153 @@
+import re
+from typing import Generator, List, Optional, Pattern, Tuple
+
+from typing_extensions import Literal
+
+OwnerTuple = Tuple[Literal["USERNAME", "TEAM", "EMAIL"], str]
+
+
+TEAM = re.compile(r"^@\S+/\S+")
+USERNAME = re.compile(r"^@\S+")
+EMAIL = re.compile(r"^\S+@\S+")
+MASK = "/" * 20
+
+
+def path_to_regex(pattern: str) -> Pattern[str]:
```

Comment:
Is it just this function that's borrowed from the codeowners tool? The CodeOwners class kinda looks like it's from there too.

Reply:
Yep, that's incorrect. I've moved the copyright text at the top of the file in: a4b3f325abbd8625219aef0fb2a50f1fdfc44190